core/chains/solana/monitor/balance.go

```diff
@@ -21,7 +20,7 @@ type Config interface {
 
 // Keystore provides the keys to be monitored.
 type Keystore interface {
-	GetAll() ([]solkey.Key, error)
+	Keys(ctx context.Context) ([]string, error)
```

Comment:
nit: `Keys` sounds like it's returning something you can sign with like `GetAll` ? maybe `Addresses` is better 🤷🏼

Reply:
WDYT about `accounts`? b/c the `Sign` api already uses `account`.

Reply:
https://github.com/smartcontractkit/chainlink-relay/pull/102